utils.py

```diff
@@ -291,3 +291,8 @@ def get_combination_matrices_summary(project_id, lib1_id, lib2_id, percentiles):
         .filter(models.MatrixResult.lib2_id == lib2_id)
     return pd.read_sql(query.statement, session.get_bind())\
         .describe(percentiles=percentiles)
+
+def tissues():
```

Comment:
the name 'tissues' is not descriptive enough for this function, please change to something like 'get_all_tissues()'

Reply:
```suggestion
def get_all_tissues():
```